WooCommerce/Classes/ViewRelated/Products/ProductsViewController.xib

```diff
@@ -42,9 +42,6 @@
                         </view>
                         <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="1mE-SE-uK9">
                             <rect key="frame" x="0.0" y="51" width="414" height="797"/>
-                            <userDefinedRuntimeAttributes>
-                                <userDefinedRuntimeAttribute type="string" keyPath="accessibilityIdentifier" value="orders-table-view"/>
-                            </userDefinedRuntimeAttributes>
```

Comment:
I'm not sure when this accessibility identifier has been wrongly set for the products list, but only just now that a UI test fails due to this. Removing the identifier fixes it.